cmu_graphics/cmu_graphics.py

```diff
@@ -679,5 +685,18 @@ def print_debug_info():
 accentCombinations = sli.accentCombinations
 t = sli.t
 
+SHAPES_CREATED = 0
+MAINLOOP_RUN = False
+
+# Checks to see if a user created shapes but did not call
+# cmu_graphics.run()
+def check_for_exit_without_run():
+    global SHAPES_CREATED, MAINLOOP_RUN
+
+    # The app's top level group is created even if the user creates no
+    # shapes on their own
+    if SHAPES_CREATED > 1 and not MAINLOOP_RUN:
+        print("To run your animation, add cmu_graphics.run() to the bottom of your file")
```

Comment:
Maybe add some ASCII art or a couple newlines before this statement so that it is more visible?